src/System.ComponentModel.TypeConverter/tests/SampleClasses.cs

```diff
@@ -82,13 +82,7 @@ public MyTypeListConverter(Type[] types)
         }
     }
 
-#if FUNCTIONAL_TESTS
-    [TypeConverter("System.ComponentModel.Tests.BaseClassConverter, System.ComponentModel.TypeConverter.Tests, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null")]
```

Comment:
we need to investigate this one more to understand why couldn't load System.ComponentModel.TypeConverter.Tests. this could be assembly binding issue?
cc @weshaggard @ericstj

---
In reply to: [106522882](https://github.com/dotnet/corefx/pull/17206#discussion_r106522882) [](ancestors = 106522882)